### DISCUSSION TITLE
Issues with Python and terminal emulators

### DISCUSSION THREAD

**Discussion by IsmarCostabile** [Kidel/Kathara#312] [General]
On my m1 pro mac book starting: "kathara vstart -n pc1 --eth 0:A" from "wezTerm"

with katharas terminal emulator choice set to either iTerm or default macOS terminal.

results in the following error:
"clear && "/usr/local/bin/kathara" connect -v -l pc1 && exit
❯  clear && "/usr/local/bin/kathara" connect -v -l pc1 && exit
[PYI-12714:ERROR] Failed to load Python shared library '/var/folders/j9/dzdwyry57s17bv2sd_qhhh_r0000gn/T/_MEIp4VJ6B/Python': dlopen: dlopen(/var/folders/j9/dzdwyry57s17bv2sd_qhhh_r0000gn/T/_MEIp4VJ6B/Python, 0x000A): tried: '/var/folders/j9/dzdwyry57s17bv2sd_qhhh_r0000gn/T/_MEIp4VJ6B/Python' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/var/folders/j9/dzdwyry57s17bv2sd_qhhh_r0000gn/T/_MEIp4VJ6B/Python' (no such file), '/var/folders/j9/dzdwyry57s17bv2sd_qhhh_r0000gn/T/_MEIp4VJ6B/Python' (no such file)"

this issue does not occur when using tmux as the choice of terminal emulator.

I can still manually execute "kathara" connect -v -l pc1" and have access to the shell.

**Comment by Skazza94** [comment]
Hi @IsmarCostabile,
the problem was related to PyInstaller not properly handling hidden imports.

We released a new package that you can download [here](https://github.com/KatharaFramework/Kathara/releases/download/3.7.7/Kathara-macos-installer-arm64-3.7.7-FIX.pkg).

Please, let us know if the problem solves!

Cheers,
Mariano.

**Comment by IsmarCostabile** [comment]
Hello, yes issue now appears to be fixed

**Comment by rolbk** [comment]
Fixed the issue for me as well, I used v3.7.7 from the homebrew cask before.
macOS 15.0.1 on an M1 Max
